pykrige/compat.py

```diff
@@ -11,12 +10,6 @@
     from sklearn.model_selection import train_test_split
 
     SKLEARN_INSTALLED = True
-    arg_spec = inspect.getfullargspec(GridSearchCV)[0]
-    # https://stackoverflow.com/a/56618067/6696397
-    if "return_train_score" in arg_spec:
-        GridSearchCV = partial(GridSearchCV, return_train_score=True)
-    if "iid" in arg_spec:
-        GridSearchCV = partial(GridSearchCV, iid=False)
```

Comment:
iid is deprecated and will be removed in the next version.

Reply:
Since when it was deprecated?

Reply:
> https://scikit-learn.org/0.20/modules/generated/sklearn.model_selection.GridSearchCV.html

Changed in version 0.20: Parameter iid will change from True to False by default in version 0.22, and will be removed in 0.24.

> https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html

Deprecated since version 0.22: Parameter iid is deprecated in 0.22 and will be removed in 0.24

It's False by default for recent versions, so there is no need to set it to false. 0.20 was released in 2018. I don't think we need to support 0.19 (or maybe even more than a couple of scikit-learn releases really).

Reply:
I am fine with that.